README.md

```diff
@@ -82,6 +82,14 @@ Restart the server from a clean state by
 make reboot
 ```
 
+If "make run" fails, please try to recover state database by
```

Comment:
"If "make run" fails" is too generic, need a bit more detail about the symptom, e.g.,
case 1: blockstore messed up while statestore is good
case 2: statestore messed up while blockstore is good
case 3: both messed up